manifests/config.pp

```diff
@@ -3,14 +3,21 @@
 # Controls the contents of falco.yaml and sets up log rotate, if needed
 #
 class falco::config inherits falco {
-  file { '/etc/falco/falco.yaml':
-    ensure  => file,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    content => template('falco/falco.yaml.erb'),
-    require => Class['falco::install'],
-    notify  => Service['falco'],
+  file {
+    default:
+      ensure  => file,
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      require => Class['falco::install'],
+      notify  => Service['falco'],
+      ;
+    '/etc/falco/falco.yaml':
+      content => template('falco/falco.yaml.erb'),
+      ;
+    '/etc/falco/falco_rules.local.yaml':
```

Comment:
If `local_rules` hasn't been set, do you want to manage this file at all (and remove everything that the installer or a user had manually put in it)?  If you always want to manage this file, I'd probably not make the type `Optional`, but default it to empty hash `{}` instead.